webpack.common.js

```diff
@@ -96,18 +98,38 @@ async function common () {
       alias: {
         '_': path.join(__dirname, 'src'),
       },
-      extensions: ['.js', '.jsx', '*'],
+      extensions: ['.js', '.jsx', '.json'],
+      fallback: {
+        module: false,
+        dgram: false,
+        dns: false,
+        fs: false,
+        http2: false,
+        net: false,
+        tls: false,
+        child_process: false,
+      },
     },
 
     output: {
-      filename: '[name].js',
       path: path.resolve(__dirname, 'dist/ui-extensions-resources'),
+      clean: true,
+
+      filename: 'js/[name].js',
+      chunkFilename: 'js/[name].[id].chunk.js',
 
       // UI plugin resources are served through Engine
       publicPath: '/ovirt-engine/webadmin/plugin/ui-extensions/',
     },
 
     optimization: {
+      // Keep the vendor chunk id consistent between builds
```

Comment:
Not sure if we should have any optimization here. It will get disabled or modified heavily anyway when running in development mode. In Vm Portal we have it only in prod mode.
The same applies to chunks - they seem typical prod mode setting to me.